content/Watsonx/WatsonxAI/102.es.md

```diff
@@ -0,0 +1,195 @@
+    
+         - En el modo **Codicioso**, el modelo selecciona los tokens de mayor probabilidad en cada paso de la decodificación, y un modelo es menos creativo en ese modo.
+    
+         - En el modo **Muestreo**, el modelo elige el siguiente token de un grupo de los siguientes tokens más probables, por lo que hay más creatividad, pero también un mayor riesgo de que el resultado no tenga sentido.
+
+
+     - **b. Temperatura**: un número de coma flotante que oscila entre 0,0 (que hace que el modelo funcione de la misma manera que si se seleccionara la decodificación Greedy) y 2,00 (que es la máxima creatividad).
+    
+         - Cuanto mayor sea el valor de **Temperatura**, más “creativo” será el modelo.
+    
+         - El valor predeterminado suele ser 0,7, pero en este caso se establece en 0,8. Esto significa que al modelo se le permite cierta creatividad.
+
+
+     - **C. Top P (muestreo de núcleos)**: un número de coma flotante que oscila entre 0,0 y 1,0. En un nivel alto, al generar una finalización, un modelo calcula la probabilidad de la siguiente palabra en función de todas las palabras anteriores. **Top P** decide si el modelo siempre elegirá el resultado más probable o permitirá más aleatoriedad para las siguientes palabras. El muestreo de **Top P** elige entre el conjunto más pequeño posible de "siguientes" palabras cuya probabilidad acumulada excede el valor proporcionado para **Top P**.
+
+     - **d. Top K**: un número entero del 1 al 100. A diferencia de **Top P, Top K** no analiza la probabilidad de los tokens. En cambio, el modelo elige la siguiente palabra de los **Top K** tokens más probables para el siguiente resultado.
+
+         - **Top K** = 3, significa que el modelo elegirá aleatoriamente entre las 3 siguientes palabras más probables.
+
+         - Cuanto mayor sea el valor de **Top K**, más aleatorio será el resultado.
+
+     - **e. Semilla aleatoria**: un número entero en el rango de 1 a 4.294.967.295. En el modo de muestreo, si todo lo demás permanece igual, la actualización de la semilla aleatoria producirá resultados diferentes.
+
+         - Si la semilla aleatoria sigue siendo la misma, debería obtener los mismos resultados, por eso dejamos el valor igual para estos laboratorios.
+
+         - En resumen, la semilla aleatoria es útil para la replicabilidad de los experimentos.
+
+     - **f. Penalización por repetición**: un valor entre 1 y 2 (un valor de 1 permite la repetición y 2 la prohíbe). Esto se utiliza para contrarrestar la tendencia de un modelo a repetir el texto del mensaje palabra por palabra o quedarse atascado en un bucle al generar el mismo resultado.
+
+     -**g. Secuencias de parada**: secuencias de caracteres (texto, caracteres especiales, retorno de carro, etc.) utilizadas como indicador de parada por el modelo. La secuencia de parada en sí seguirá incluida en el resultado del modelo, pero esa será la última parte del resultado.
```

Comment:
![Screenshot 2023-10-11 at 3 40 45 PM](https://github.com/ibm-build-lab/VAD-VAR-Workshop/assets/7364498/ff9f6ac0-bc16-41ea-9312-e5e4f72f9d65)

A few abnormalities here.  
On line 69, we need a space between the - and **g so it gets interpreted as a bullet point
On line 71, we need to remove the space between ** and h, so it will show up as bold text. 
On line 73 the header should be on the far left side, rather than tabbed over.

Reply:
There is also potentially an issue when translation of values that correspond to items in the screenshots.  In several sections we include a screenshot (which is in english) and then describe the places on the screenshot, those labels are being translated.  The a, b, c...etc labeling should be enough to match them up in these cases.   But there are some other cases where we may not have such alphanumeric labels.   

One pertinent question:
Would a spanish speaking person most likely be viewing the UI in english or spanish?  If they are likely to view the page in spanish, either because watsonx is by default localized (I'm not sure if this is the case), or if a spanish speaker would likely use a browser plugin to translate the page, then maybe it's better if we allow translation.  

The 5 question quiz that the BP needs to pass would be in english, unless of course the same browser plugin was used to translate the quiz.

I think it may be best to let this in and get Hans/David to weigh in on what would make more sense for translating UI labels in the instructions.